<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWindow</class>
 <widget class="QMainWindow" name="HistoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/Logo.png</normaloff>:/images/Logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-2</y>
      <width>844</width>
      <height>484</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/BlankBackground.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Schoolbook L</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 115, 115);
</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>350</y>
      <width>310</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Date:&lt;br/&gt;Beer:&lt;br/&gt;Ounces:&lt;br/&gt;Rounded:&lt;br/&gt;Price:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Schoolbook L</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 115, 115);</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>66</width>
      <height>66</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/CloseButton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="previewLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>260</width>
      <height>190</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="usersComboBox">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>310</y>
      <width>321</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox QAbstractItemView
{
	background-color: rgb(120, 120, 120);
	selection-background-color: rgb(90, 90, 90);
}
QComboBox
{
	background-color: rgb(120, 120, 120);
	min-height:34px;
	selection-background-color: rgb(90, 90, 90);
}
QScrollBar:vertical
{
	width:50px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(80, 80, 80, 255), stop:1 rgba(105, 105, 105, 255));
}
QScrollArea QWidget
{
	background-color: rgb(110, 110, 110);
}
QButton QWidget
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(91, 91, 91, 255), stop:1 rgba(155, 155, 155, 255));
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="snapIndexLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>30</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>1/4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Schoolbook L</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="StaticScrollArea" name="scrollPane">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>360</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>350</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(173, 173, 173);</string>
    </property>
    <property name="text">
     <string>History</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imageBackground">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>260</width>
      <height>190</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>imageBackground</zorder>
   <zorder>prevButton</zorder>
   <zorder>infoLabel</zorder>
   <zorder>nextButton</zorder>
   <zorder>closeButton</zorder>
   <zorder>previewLabel</zorder>
   <zorder>usersComboBox</zorder>
   <zorder>snapIndexLabel</zorder>
   <zorder>deleteButton</zorder>
   <zorder>scrollPane</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StaticScrollArea</class>
   <extends>QFrame</extends>
   <header>widgets/staticscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
